packages/instant/src/containers/latest_error.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { connect } from 'react-redux';
+
+import { SlidingError } from '../components/sliding_error';
+import { State } from '../redux/reducer';
+import { errorDescription } from '../util/error_description';
+
+export interface LatestErrorComponentProps {
+    assetData?: string;
+    latestError?: any;
```

Comment:
Per convo w/ @fragosti , we could had two options:

1) When error is "cleared" (after timeout), set `currentError` to be undefined. This would have been nice because it's the most straightforward representation of the current state, but animating the unmounting of the component would either be pretty hairy or involve introducing a new dependency.

2) Keep the error around in `latestError` but set `latestErrorDismissed` to `true`, and then render the sliding down error message.  I don't love keeping the last error around but treating it as "dismissed", but it makes the animation logic simple.

After discussing, I went with #2 for the drawbacks listed in #1.  Open to feedback though.